### PR TITLE
Beautify services

### PR DESCRIPTION
**Pull request by ghost**
This gives each service an individual line and give a little indent between the service and the active/inactive output.

### CODE REVIEW

**Comment by HermannBjorgvin** [comment]
That takes up too much space, the reason the services message is compacted is because otherwise it would take up like 10-20 lines. If you want to add this you'll need to add a "servicesWidth" field to config.json and pull the value you need from that. But this won't be the default since it's just too big.

File: scripts/services.sh [hunk 1]
```diff
@@ -33,7 +33,7 @@ done
 
 # Maximum column width
 #width=$((49-2))
-width=$((64-2))
+width=$((14-2))
```
Comment (HermannBjorgvin) [inline scripts/services.sh:36]: Too inefficient on terminal real estate

**Comment by HermannBjorgvin** [comment]
Maybe create another version of the script? Append the filename to say -long or something? I was also thinking yesterday it might be cool to add a list of services that have failed to the list automatically

**Comment by ghost** [comment]
Good idea but I'll work on that later.  I have another project I'm on with right now but this really piqued my interest on reddit.
Hope to iron out the bugs with everyone.

**Comment by ghost** [comment]
I'm going to wait and see what happens with the branch that is moving everything to bash.  I have a feeling that is going to take precedent at the moment.

**Pull request closed without merging**